docs/releases/2.0.txt

```diff
@@ -236,6 +236,14 @@ Models
   from the database. For databases that don't support server-side cursors, it
   controls the number of results Django fetches from the database adapter.
 
+Aggregation
+~~~~~~~~~~~
+
+* Built in aggregates now accept a ``filter`` parameter that can be used to
+  exclude elements from an aggregate based on a condition. On backends that
+  support it the SQL 2003 ``FILTER WHERE`` syntax is used, otherwise a standard
```

Comment:
I think it's good to note somewhere in the documentation, maybe not in the release notes however.